pkg/arvo/lib/push-hook.hoon

```diff
@@ -378,26 +412,51 @@
     [%pass wire %agent [our.bowl store-name.config] %watch store-path.config]
   ::
   ++  push-updates
-    |=  =vase
+    |=  =cage
     ^-  (list card:agent:gall)
-    =/  rids=(list resource)  (resource-for-update vase)
+    =/  rids=(list resource)  (resource-for-update q.cage)
     =|  cards=(list card:agent:gall)
     |-
```

Comment:
I believe I changed this out to be a `+murn` in my PR, and that still seems prudent, given that it's jetted, whereas manual loops are not, and are more bespoke than a simple `+turn` with a `+zing` at the end or a `+murn`.